.gitignore

```diff
@@ -20,3 +20,6 @@ access.log
 .vertx
 **openapi.json
 fileStore/failedBrokers.txt
+cruise-control/src/main/resources/webroot/.openapi-generator-ignore
```

Comment:
Just curious do we want to add all contents in webroot or we shouldn't do that? I'm not familiar with that part.

Reply:
The other contents are not generated, they are needed for the UI. So we don't want to add them here.